products/protractor.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Protractor
+category: framework
+tags: google javascript-runtime test
+iconSlug: protractor
+permalink: /protractor
+versionCommand: npm list protractor
+
+releaseDateColumn: true
+activeSupportColumn: Community Support
+extendedSupportColumn: Commercial Support
+eolColumn: false
+
+auto:
+-   npm: protractor
+
+identifiers:
+-   purl: pkg:npm/protractor
+
+releases:
+-   releaseCycle: "7.0"
+    support: 2023-08-31
+    eol: true
```

Comment:
```suggestion
    eol: 2023-08-31
```

See previous comment. Same for all versions.

Reply:
Updated in all versions.